src/pages/iou/request/step/IOURequestStepDistance.js

```diff
@@ -211,10 +224,10 @@ function IOURequestStepDistance({
                 </View>
                 <View style={[styles.w100, styles.pt2]}>
                     {/* Show error message if there is route error or there are less than 2 routes and user has tried submitting, */}
-                    {((hasError && _.size(validatedWaypoints) < 2) || (_.keys(waypoints).length > 2 && _.size(validatedWaypoints) !== _.keys(waypoints).length) || hasRouteError) && (
+                    {getError && (
```

Comment:
I don't mean we use getError here. Let's use `hasError` variable or using conditions directly
```
(duplicateWaypointsError || atLeastTwoDifferentWaypointsError || hasRouteError || isLoadingRoute || isLoading)
```
 and remove hasError variable